var/spack/repos/builtin/packages/esmf/esmf890b08llvm20.patch

```diff
@@ -0,0 +1,20 @@
+--- a/build_config/Linux.llvm.default/build_rules.mk
++++ b/build_config/Linux.llvm.default/build_rules.mk
+@@ -6,7 +6,7 @@
+ ############################################################
+ # Default compiler setting.
+ #
+-ESMF_F90DEFAULT         = flang
```

Comment:
This change is strictly speaking not required, but I don't want to make the effort and revert this given that the patch will be removed when the next beta release is added and b08 is removed.